events/Process/process_tools.js

```diff
@@ -0,0 +1,133 @@
+const config = require("../../config");
+
+module.exports = {
+  Translate: async (text = "", lang = "", allLowerCase = false) => {
+    let output;
+    let wait_time = config.app?.Translate_Timeout;
+
+    let reg = /<([^>]+)>/g;
+
+    if (!translate) {
+      console.warn("❌ No translation module detected! ❌");
+      output = getUnchangedText(text);
+      return output;
+    }
+
+    // Apparently doing this searches it without crashing. Damn
+    !lang ? (lang = config.app?.lang) : (lang = lang);
+
+    if (!text || !lang)
+      throw new Error(
+        "❌ You must provide a text and a language code to translate! ❌"
+      );
+
+    if (lang === "en") {
+      output = getUnchangedText(text);
+    } else {
+      const arrayStr = text.split(reg);
+      const translatedArray = await Promise.all(
+        arrayStr.map(async (str, index) => {
+          if (index % 2 == 0) {
+            if (verifyLang(lang)) {
+              try {
+                let Tranlate_buff;
+
+                if(wait_time){
+                  const timeout = new Promise((resolve, reject) => {
+                    setTimeout(() => {
+                      reject(new Error('❗ TimeoutRaisedError: The Translation took too long to complete! Skipping... ❗'))}, wait_time);
+                  })
+                  Tranlate_buff = await Promise.race([translate(str, lang), timeout]);
+                } else{
+                  Tranlate_buff = await translate(str, lang);
+                }
+
+                if(!allLowerCase) return Tranlate_buff;
+                return Tranlate_buff.toLowerCase();
+              } catch (e) {
+                return getUnchangedText(str);
+              }
+            } else {
+              console.clear()
+
+              const module = await import("chalk");
+              chalk = module.default || module;
+
+            genConfigError(chalk, 'app', 'lang', 
+            `❌ An invalid language was inserted in the config file. Please check the language code! ❌
+            \t\t\tchange the language code in the config.js file\n`);  
+            }
+          } else {
+            return getUnchangedText(str);
+          }
+        })
+      );
+      output = translatedArray.join(" ");
+    }
+
+    return output;
+  },
+
+  GetTranslationModule: async () => {
+    try {
+      const module = await import("translate");
```

Comment:
use require keep it like the rest of the code space

Reply:
It is an ES module